app/scss/components/_grid.scss

```diff
@@ -166,7 +166,14 @@
   @include flex(0, 0, 90%);
   max-width: 90%;
 }
-
+.left {
```

Comment:
It's like everyone contributing is just not reading through the README and CONTRIBUTING guide and that's too bad